### ISSUE TITLE
 CI and Settings buttons do not work

### ISSUE THREAD

**Issue by ValentinaPC** [mozilla-mobile/mozilla-vpn-client#1159]
#### VPN version:
- 2.3.0 (2.202106080944);

#### Affected Platforms:
- Linux OS;
- Windows 10;

#### Prerequisites:
- Have one FxA account with subscription;
- Have the VPN open and ON;

#### Steps to reproduce:
1. Click the “i” connection information button;
2. Click the Settings gear icon;

#### Expected result:
- Both options work as expected.

#### Actual result:
- The options do not work;

#### Notes:
- Accessing Settings is possible from the tray menu.

**Comment by oskirby** [comment]
I think this is due to a recent change in the glean_parser package. I have been able to work around it in my development environment by downgrading my glean_parser package to 3.4.0. The start of the problems seems to have to do with the glean imports failing, after which any calls to glean by the QML will result in an error and sometimes prevent widgets from working.

```
[08.06.2021 07:38:54.910] Warning: qrc:/glean/generated/pings.js:13: TypeError: Cannot read property '_private' of undefined (pings.js:13)
[08.06.2021 07:38:54.910] Warning: qrc:/glean/generated/sample.js:13: TypeError: Cannot read property '_private' of undefined (sample.js:13)
```

**Comment by oskirby** [comment]
It seems that the problem is due to a version mismatch between glean_parser and our glean.js version, and we need to update glean.js to version v0.15.

**Comment by bakulf** [comment]
is this in prod?

**Comment by ValentinaPC** [comment]
> 
> 
> is this in prod?

no no, stage (last build from GitHub)

**Comment by bakulf** [comment]
This is almost done. It should be fixed soon.

**Comment by ValentinaPC** [comment]
Verified as fixed on the last VPN 2.4 build, across platforms.
Both buttons work as expected.
